tests/stream/test_stream_smoke/0028_replay_stream.yaml

```diff
@@ -53,12 +53,12 @@ tests:
           - client: python
             query_type: stream
             terminate: auto
-            query_end_timer: 20
+            query_end_timer: 10
             query_id: '2900'
             wait: 1
             query: with cte as (select id, value, date_diff('millisecond', to_datetime64(lag(_tp_append_time, 1)/1000, 3, 'UTC'), to_datetime64(_tp_append_time/1000, 3, 'UTC')) as insert_interval,
-                    date_diff('millisecond',lag(now64(), 1), now64()) as output_time_diff, _tp_time from test29_stream where _tp_time > earliest_timestamp() settings replay_speed=1)
-                    select id, value, abs(insert_interval-2000) < 200, abs(insert_interval-output_time_diff) < 10 from cte;
+                    date_diff('millisecond',lag(now64(), 1), now64()) as output_time_diff, _tp_time from test29_stream where _tp_time > earliest_timestamp() settings replay_speed=1, enable_backfill_from_historical_store=false)
```

Comment:
when we reset backfill flag to false for replay in the code, we don't need explicitly set backfill flag to false here.